Project.toml

```diff
@@ -25,10 +25,10 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
 [compat]
-ColorSchemes = "3.19"
+ColorSchemes = "≥ 3.19"
```

Comment:
It's not a good idea to allow arbitrary breaking releases with `>=`. You can't ensure this package works with ColorSchemes v4, or Crayons v5 unless they're released.

A lot of Julia packages used this, and then the entire ecosystem shifts to the more restricted version. https://github.com/JuliaRegistries/General/pull/11114